selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/GolangFileEditingTest.java

```diff
@@ -117,6 +146,13 @@ public void checkCodeValidationFeature() {
     editor.goToCursorPositionVisible(13, 1);
     editor.typeTextIntoEditor(Keys.DELETE.toString());
     editor.waitAllMarkersInvisibility(ERROR);
+
+    // check adding comment code by keyboard 'Ctrl+/'
```

Comment:
`// check adding comment code by keyboard 'Ctrl+/'` > `// check code line commenting`

Reply:
I agree

Reply:
I would also move comment code verification into separate test `checkCodeCommentFeature` so that it doesn't connected to code validation feature.